dymos/transcriptions/pseudospectral/radau_pseudospectral.py

```diff
@@ -107,26 +108,15 @@ def configure_ode(self, phase):
         super(Radau, self).configure_ode(phase)
 
         grid_data = self.grid_data
-        num_input_nodes = grid_data.subset_num_nodes['state_input']
         map_input_indices_to_disc = grid_data.input_maps['state_input_to_disc']
 
         for name, options in phase.state_options.items():
-            size = np.prod(options['shape'])
-
-            src_idxs_mat = np.reshape(np.arange(size * num_input_nodes, dtype=int),
-                                      (num_input_nodes, size), order='C')
-
-            src_idxs = src_idxs_mat[map_input_indices_to_disc, :]
-
-            if options['shape'] == (1,):
```

Comment:
nice not to have to do all that anymore.

Reply:
I agree!  om.slicer is so much nicer than figuring out how to build src_indices for arbitrary shapes